Tests/PointFreeTests/__Snapshots__/UpdateProfileTests/testUpdateEmailSettings.2.Conn.txt

```diff
@@ -4,11 +4,11 @@
 ▿ Request
   POST http://localhost:8080/account
   Authorization: Basic aGVsbG86d29ybGQ=
-  Cookie: pf_session=be847b5f12f9571a96252587105c91f788704838959a4818df9555a7dbb636e09146d686873dfde259849eb38151c9f448d718ff57b50f22d67cc9c37c9acca2b8da4c3e919a230d2b2d4222dedc4e9c354d84975c9e5f532503e6be65314e300f74e793d154bf8f3c85054ae8df44717e856dc441f24a070cad129b4072a87a
+  Cookie: pf_session={"userId":"00000000-0000-0000-0000-000000000001"}
 
   name=&emailSettings[newEpisode]=on&email=
 
 ▿ Response
   Status 302 FOUND
   Location: /account
-  Set-Cookie: pf_session=2e2e083b6dbb81835625f1b89f635da6c10f6d821f655192f7f23bd3854e145c95004ce096a5a07374b35276f89f0a4cb311acee81e5bef5f30e6f85ce9f04393d1437dba175fd17c25770ffc6efd44359c0d055b2dcb9713268fb12467b92661656052b62e19e1ca7f91fd3c0488b027d9c6dc7a63e53c03f52f1940cb3bce3fed39e879e91d1bc43759e3c252b54b67b680be051288c06d8de7bba15bd04bc487619bbf513ec0c2ef538fd0b822b9226e3b707294907014ac23ceb79f9cb134874a46829e741a1bd60dbe055367c6dde218adefdf5c344bf285bd7306909fa5e09194a5bc8fb8f3918db421b88d0e4; HttpOnly; Path=/
\ No newline at end of file
+  Set-Cookie: pf_session={"flash":{"message":"We’ve sent an email to  to confirm this change.","priority":"warning"},"userId":"00000000-0000-0000-0000-000000000001"}; Path=/
```

Comment:
oh ha, this message aint right. I was doing something weird in the test, not an app bug... will fix